blockbase/sass/base/_layout.scss

```diff
@@ -21,3 +21,10 @@ body.admin-bar {
 		min-height: calc( 100vh - var(--wpadmin-bar--height) );
 	}
 }
+
+// Provided to allow a group to prevent the standard gap from being used between elements
+.gapless-group {
+	> * {
+		margin-top: 0 !important;
+	}
+}
```

Comment:
should this be in [_utility.scss](https://github.com/Automattic/themes/blob/trunk/blockbase/sass/base/_utility.scss) instead?

Reply:
I agree.  Changed.